### PR TITLE
react-accordion-component | Lucca Rizzi

### PR DESCRIPTION
**Pull request by luccarizzi**
new1

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const languages = [
```
Comment (Zhiwutian) [inline react-accordion-component/src/accordion.jsx:3]: This should be defined in index.js and passed in as props to the Accordion JSX element

File: react-accordion-component/src/accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({
+      id: event.target.id
+    });
+    const id = this.state.id;
+    if (id === event.target.id) {
+      event.target.nextSibling.className = 'd-none';
```
Comment (Zhiwutian) [inline react-accordion-component/src/accordion.jsx:36]: In React, you do not want to access elements on the DOM directly. React should always be in control of what is displayed on the page. Any changes should be done directly to the JSX which is returned from the render method.

**Review by Zhiwutian — APPROVED** [review]
some updates

File: react-accordion-component/src/accordion.jsx (hunk 1 above)
Comment (luccarizzi) [reply react-accordion-component/src/accordion.jsx]: Done.

File: react-accordion-component/src/accordion.jsx (hunk 2 above)
Comment (luccarizzi) [reply react-accordion-component/src/accordion.jsx]: Done.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**